example/src/Demo.jsx

```diff
@@ -31,6 +31,8 @@ function Demo({ fetchSwagger, status, docs, oas, oauth }) {
               'api-setting': Object.assign(extensions.defaults, oas),
             }}
             baseUrl={'/'}
+            // Uncomment this if you want to test enterprise-structured URLs
+            // baseUrl={'/child/v1.0/'}
```

Comment:
If `baseUrl` ends with a slash, then the resulting URL has a double slash in the middle of it:

```
http://localhost:9966/child/v1.0//docs/link
```

I think a concatenation is good enough, as long as we can guarantee ReadMe will only pass URLs without the slash on the end.